ibodata/lateral_profile.py

```diff
@@ -87,17 +87,17 @@ def asymmetry(self):
         Add area between mid and the nearest points if there's no 0 value in self.x
         area_left and area_right are DataSet and it propagates to result so it has to be converted to float
         """
-        area_left = np.trapz(self.y[self.x <= 0], self.x[self.x <= 0])
-        area_right = np.trapz(self.y[self.x >= 0], self.x[self.x >= 0])
+        area_left = np.trapz(x=self.x[self.x <= 0], y=self.y[self.x <= 0])
+        area_right = np.trapz(x=self.x[self.x >= 0], y=self.y[self.x >= 0])
 
         if np.argwhere(self.x == 0).size == 0:
             left_index_arr = np.argwhere(self.x < 0)
-            area_left += np.trapz(np.append(self.y[left_index_arr[left_index_arr.size - 1]], self.y_at_x(0)),
-                                  np.append(self.x[left_index_arr[left_index_arr.size - 1]], [.0]))
+            area_left += np.trapz(x=np.append(self.x[left_index_arr[left_index_arr.size - 1]], [.0]),
```

Comment:
python supports negative indexing, last item of a table `t` can be obtained by call to `t[-1]`
It looks that `left_index_arr.size` is not needed